accepted/2023/owner-author-pmui.md

```diff
@@ -0,0 +1,214 @@
+The enumerable owners will allow displaying custom UI for each owner.
+
+#### Tabs other than Browse Tab
+
+Package metadata is only retrieved on the Browse Tab, therefore, it is the only tab that currently looks up Owners from a package source.
+Other tabs will continue displaying the author.
+
+The [MultiSourcePackageFeed](https://github.com/NuGet/NuGet.Client/blob/1942c17866695ae1175ec3d8b42a6b41bcd6a9c9/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageFeed.cs#L117) queries all package sources for metadata and combines the responses.
+
+## Drawbacks
+
+<!-- Why should we not do this? -->
+N/A
+
+## Rationale and alternatives
+
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+
+### Icon with hyperlink
+
+Including an icon to further indicate that the owner in the packages list easily becomes noisy.
+![Owners from NuGet.org are shown beside each package ID with a hyperlink and a NuGet Package Icon](../../meta/resources/OwnerAuthor/ownerNuGetOrgLinks.png)
+
+### Special label for owner / author
+
+Using a prefix of `by:` with the colon in the same manner as `www.nuget.org`, we could start letting this imply Owner, and introduce a new term for Author.
+
+Only render `by:` when we have Owners from a package source. The tooltip will reflect the same information.
+![PM UI screenshot showing a package with "by:" followed by a list of Owners, and a tooltip which also reflects those Owners](../../meta/resources/OwnerAuthor/byOwnerExample.png)
+
+Render a different string, `contributors:` when we only have Authors and do not have Owners from a package source. The tooltip will reflect the same information.
+![PM UI screenshot showing a package with "contributors:" followed by the Authors metadata, and a tooltip which also reflects those Authors](../../meta/resources/OwnerAuthor/contributorsAuthorExample.png)
+
+## Prior Art
+
+<!-- What prior art, both good and bad are related to this proposal? -->
+<!-- Do other features exist in other ecosystems and what experience have their community had? -->
+<!-- What lessons from other communities can we learn from? -->
+<!-- Are there any resources that are relevant to this proposal? -->
+
+N/A
+
+## Unresolved Questions
+
+<!-- What parts of the proposal do you expect to resolve before this gets accepted? -->
+<!-- What parts of the proposal need to be resolved before the proposal is stabilized? -->
+<!-- What related issues would you consider out of scope for this proposal but can be addressed in the future? -->
+
+1. Edge case: The package source responds with a Trusted Owner, but doesn't provide URL templates for **Owner Details** nor **Package Details**.
```

Comment:
Maybe show something like (incomplete metadata from source).

Reply:
Definitely worth considering something like this, especially once more package sources support Owner.
For this first iteration, how do you feel about just making these prerequisites for showing Owner? So, if Search API returns owner, and at least one URL template is available, show Owner, otherwise keep current behavior.

Reply:
I don't feel too strongly. 
I just suggested this because I thought it was the more transparent behavior.